### DISCUSSION TITLE
How to Use WorkflowStatusListener in coductor

### DISCUSSION THREAD

**Discussion by karanbhola** [Netflix/conductor#3624] [Q&A]
Documentations are not very clear on how to properly use WorkflowStatusListener. Documentation tells about following:

- Setting the workflowStatusListenerEnabled field in your Workflow Definition to true enables notifications.

- To add a custom implementation of the Workflow Status Listener.

It is no where mentioned how can we implement this interface and how the conductor will came to know about our implementation

How can I send a notification to an external system ??

**Comment by charybr** [comment]
Hi,

I have created a module called Webhook in conductor-community repo that uses WorkflowStatusListener. Please have a look at this PR
https://github.com/Netflix/conductor-community/pull/222

Basically we need to add below config to make webhook to work it
conductor.workflow-status-listener.type=workflow_publisher
conductor.webhook.notification.url=
conductor.webhook.notification.endpointWorkflow=workflow/WorkflowNotifications
